MiniSim/Views/ParametersTable/ParametersTableForm.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ParametersTableForm.swift
+//  MiniSim
+//
+//  Created by Jan Jaworski on 27/10/2023.
+//
+
+import SwiftUI
+import SymbolPicker
+import CodeEditor
+
+
+struct ParametersTableForm: View {
+    var parameter: Parameter?
+    var allParameters: [Parameter]
+    var onSubmit: (_ parameter: Parameter, _ prevParameter: Parameter?) -> Void
+    
+    @Environment(\.dismiss) var dismiss
+    @Environment(\.colorScheme) var colorScheme
+    
+    @StateObject private var viewModel: ViewModel = ViewModel()
+    
+    var body: some View {
+        Form {
+            TextField("Title", text: $viewModel.title)
+            HStack(alignment: .top) {
+                CodeEditor(
+                    source: $viewModel.command,
+                    language: .bash,
+                    theme: colorScheme == .dark ? .atelierSavannaDark : .atelierSavannaLight,
+                    flags: [.selectable, .editable, .smartIndent]
+                )
+                .cornerRadius(6)
+            }
+            Toggle("Enabled", isOn: $viewModel.enabled)
+            .help("Determines if command is enabled.")
+            .toggleStyle(.switch)
+            
+            HStack {
+                HStack {
+                    Button("Cancel") {
+                        dismiss()
+                        viewModel.clearForm()
+                    }
+                }
+                Spacer()
+                Button(parameter != nil ? "Update" : "Add") {
+                    onSubmit(
+                        Parameter(
+                            title: viewModel.title,
+                            command: viewModel.command,
+                            enabled: viewModel.enabled
+                        ),
+                        parameter
+                    )
+                    viewModel.clearForm()
+                }
+                .buttonStyle(.borderedProminent)
+                .disabled(viewModel.disableForm)
+                .keyboardShortcut("s", modifiers: .command)
+            }
+            .padding(.top)
+        }
+        .padding()
+        .frame(minWidth: 550, minHeight: 400)
```

Comment:
Let's change the height to 250 as user is expected to only input the parameter like (`--no-audio`)